xdem/spstats.py

```diff
@@ -0,0 +1,507 @@
+"""
+xdem.spstats provides tools to use spatial statistics for elevation change data
+"""
+from __future__ import annotations
+import os
+import numpy as np
+import math as m
+import skgstat as skg
+from scipy import integrate
+import multiprocessing as mp
+import pandas as pd
+import ogr
+from functools import partial
+
+def get_empirical_variogram(dh: np.ndarray,coords: np.ndarray, **kwargs) -> pd.DataFrame:
+    """
+    Get empirical variogram from skgstat.Variogram object
+
+    :param dh: elevation differences
+    :param coords: coordinates
+    :return: empirical variogram (variance, lags, counts)
```

Comment:
I feel nitpicky now, but isn't the established syntax `:returns:`? I might be wrong, and I don't care that much either!

Reply:
this is the syntax that PyCharm pops out automatically when I start a function description!